pyccel/codegen/wrapper/cuda_to_c_wrapper.py

```diff
@@ -0,0 +1,82 @@
+# coding: utf-8
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
+"""
+Module describing the code-wrapping class : CudaToPythonWrapper
+which creates an interface exposing Cuda code to C.
+"""
+
+from pyccel.ast.bind_c      import BindCModule
+from pyccel.errors.errors   import Errors
+from pyccel.ast.bind_c      import BindCVariable
+from .wrapper               import Wrapper
+
+errors = Errors()
+
+class CudaToCWrapper(Wrapper):
+    """
+    Class for creating a wrapper exposing Fortran code to C.
+
+    A class which provides all necessary functions for wrapping different AST
+    objects such that the resulting AST is C-compatible. This new AST is
+    printed as an intermediary layer.
```

Comment:
Cuda is often compatible with C out of the box. Please add a sentence giving an example where the wrapper is necessary

Reply:
We added the wrapper for CUDA functionality, but so far, it only handles the base cases. I think I should change the document to something like this: 'While CUDA is typically compatible with C by default, this wrapper becomes necessary in scenarios where specific adaptations or modifications are required to ensure seamless integration with C.' In the future, when you add kernel and device, then I will add good examples. What do you think?

Reply:
That sounds good to me

Reply:
fixed d48329c

Reply:
my bad df854cc

Reply:
Missing period at the end of the sentence

Reply:
??

Reply:
there is a period at the end of the sentence ??

Reply:
https://www.grammarly.com/blog/period/

In English there is a full stop at the end of every sentence. It seems to be missing in your docstring

Reply:
fixed 064a25a